src/main/java/mchorse/vanilla_pack/morphs/BlockMorph.java

```diff
@@ -198,12 +198,12 @@ public void toNBT(NBTTagCompound tag)
     @Override
     public float getWidth(EntityLivingBase target)
     {
-        return 1;
+        return 0.99F;
     }
 
     @Override
     public float getHeight(EntityLivingBase target)
     {
-        return 1;
+        return 0.99F;
```

Comment:
I took a look on the whole PR, and it's looking good! I have few concerns with NBT property names having `camelCase` instead of `PascalCase` (which is kind of a standard), but other than that it looks great!

Also thank you very much for making this work without a coremod :+1: 

Besides that, why did you change it from `1` to `0.99F`? I'm just wondering. Last time I checked, it worked perfectly when you needed to get inside of the block.

Reply:
The NBT change is easy enough.

For the change to 0.99F, it turns out that you actually use 0.99F in updateSize(). I'll push a commit so it actually references those functions, rather than being hardcoded.